src/app/view/timestamp-editor/timestamp-editor.component.ts

```diff
@@ -98,7 +98,7 @@ export class TimestampEditorComponent implements OnInit {
       vesselPositionLatitude: new FormControl(null, [Validators.pattern("^[0-9.]*$"), Validators.maxLength(10)]),
       milesToDestinationPort: new FormControl(null, [Validators.pattern('^[0-9]+(.[0-9]?)?$')]),
       remark: new FormControl(null),
-      delayCode: new FormControl({ value: '' }),
+      delayCodes$: new FormControl({ value: '' }),
```

Comment:
This is not an observable (but a `FormControl`), so I do not think this should be have a `$` in its name.

Reply:
That's a typo, will fix it